sphinx_gallery/gen_rst.py

```diff
@@ -527,7 +531,7 @@ def execute_script(code_block, example_globals, image_path, fig_count,
 
         # Breaks build on first example error
 
-        if gallery_conf['abort_on_example_error']:
+        if gallery_conf.get('abort_on_example_error', True):
```

Comment:
Yeah I agree about keeping them in one place. I'll add a comment that it should be unified if possible.